security-analysis/security-analysis-api/src/main/java/com/powsybl/security/SecurityAnalysisResultWithLog.java

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) 2019, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package com.powsybl.security;
+
+import java.util.Objects;
+import java.util.Optional;
+
+/**
+ * @author Yichen TANG <yichen.tang at rte-france.com>
+ */
+public class SecurityAnalysisResultWithLog {
+
+    private final SecurityAnalysisResult result;
+
+    private byte[] logBytes;
```

Comment:
It is mandatory here to add some javadoc to explain what those bytes are: the API user must not have to go into the implementation code to understand what it is.